lib/srv/uacc/uacc_linux.go

```diff
@@ -195,6 +182,24 @@ func Close(utmpPath, wtmpPath string, tty *os.File) error {
 	}
 }
 
+// getDefaultPaths sets the default paths for utmp and wtmp files if passed empty.
+func getDefaultPaths(utmpPath, wtmpPath string) (string, string) {
+	if utmpPath == "" {
+		utmpPath = utmpFilePath
+	}
+
+	if wtmpPath == "" {
+		// Check where wtmp is located.
+		if utils.FileExists(wtmpFilePath) {
+			wtmpPath = wtmpFilePath
+		} else {
+			wtmpPath = wtmpAltFilePath
```

Comment:
Does `wtmpAltFilePath` always exist? And won't we get the same issue if it doesn't?

Reply:
If it doesn't exist, this function should not be called. Out logic in reexec takes this into account. I can refactor the code to make it more explicit.

Reply:
I think a comment would suffice in that case.